app/views/provider_interface/users/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :browser_title, t('page_titles.provider.users') %>
+
+<% content_for :before_content do %>
+  <%= breadcrumbs({
+    t('page_titles.provider.organisation_settings') => provider_interface_organisation_settings_path,
+    t('page_titles.provider.users') => provider_interface_organisation_settings_organisation_users_path(@provider),
+    @provider_user.full_name => nil,
+  }) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @provider.name %></span>
+    <h1 class="govuk-heading-l"><%= @provider_user.full_name %></h1>
+    <h2 class="govuk-heading-m">Personal details</h2>
+    <%= render SummaryListComponent.new(
+      rows: [
+        { key: 'First name', value: @provider_user.first_name },
+        { key: 'Last name', value: @provider_user.last_name },
+        { key: 'Email address', value: @provider_user.email_address },
+      ],
+    ) %>
+    <h2 class="govuk-heading-m">User permissions</h2>
+    <p class="govuk-body"><%= t('provider_relationship_permissions.view_applications_explanation') %></p>
+    <%= render ProviderInterface::UserPermissionSummaryComponent.new(provider_user: @provider_user, provider: @provider, editable: @current_user_can_manage_users) %>
```

Comment:
Note that this component currently has the old URL for the change links. We will fix this when we do the edit flow